src/homework11/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,59 @@
+package homework11;
+import java.util.HashMap;
+import java.util.Map;
+
+//1. Створити клас CoffeeOrderBoard. Клас є цифровим поданням черги замовлень у кав'ярні.
+//Цей клас містить список замовлень.
+public class CoffeeOrderBoard {
+
+    public final Map<Integer, Order> orderBoard = new HashMap<>();
```

Comment:
Are you sure that `Map` (or `HashMap` particularly) fit the tasks it has to do? It may be better to use another data structure type.